lib/pbench/test/unit/server/test_requests.py

```diff
@@ -77,97 +100,150 @@ def test_json_object(client, caplog, server_config):
 
 class TestUpload:
     @staticmethod
-    def test_missing_filename_header_upload(client, caplog, server_config):
-        expected_message = (
-            "Missing filename header, "
-            "POST operation requires a filename header to name the uploaded file"
-        )
+    def test_missing_authorization_header(client, caplog, server_config):
         response = client.put(
-            f"{server_config.rest_uri}/upload/ctrl/{socket.gethostname()}"
+            f"{server_config.rest_uri}/upload/ctrl/{socket.gethostname()}",
+            headers={"filename": "f.tar.xz"},
```

Comment:
yeah `dict()` doesn't accept any key that has hyphen in it like `Content-MD5` thats why I used dictionary comprehension syntax, I guess I should use it everywhere instead of `Dict`

Reply:
`dict()` will accept an "iterable" (or a mapping) in lieu of keyword arguments, and, apparently, you can even mix them, so
```
            headers=dict([("filename", "f.tar.xz")]),
```
or
```
            headers=dict((("filename", "f.tar.xz"),))
```
_would_ work here, but I find the comprehension syntax to be much more flexible and compact, so I would recommend it.